bindgen/codegen/helpers.rs

```diff
@@ -352,4 +352,30 @@ pub(crate) mod ast_ty {
             })
             .collect()
     }
+
+    pub(crate) fn to_ptr(
+        ctx: &BindgenContext,
+        ty: syn::Type,
+        is_const: bool,
+    ) -> syn::Type {
+        if is_const {
+            if let Some(wrapper) =
+                ctx.options().custom_const_pointer_type.as_ref()
+            {
+                let wrapper = ctx.rust_ident_raw(wrapper);
+                syn::parse_quote! { #wrapper<#ty> }
+            } else {
+                syn::parse_quote! { *const #ty }
+            }
+        } else {
+            if let Some(wrapper) =
+                ctx.options().custom_mut_pointer_type.as_ref()
+            {
+                let wrapper = ctx.rust_ident_raw(wrapper);
+                syn::parse_quote! { #wrapper<#ty> }
+            } else {
+                syn::parse_quote! { *mut #ty }
+            }
+        }
```

Comment:
I think this may be easier to understand as a match, roughly

```suggestion
        match (is_const, ctx.options().custom_const_pointer_type.as_ref()) {
            (true, Some(wrapper) => {
                let wrapper = ctx.rust_ident_raw(wrapper);
                syn::parse_quote! { #wrapper<#ty> }
            }
            (true, None) => { ... }
            (false, Some(wrapper) => { ... }
            (false, None) => { ... }
        }
```

Reply:
That doesn't work because you need to look at custom_mut_pointer_type too.

Reply:
Ah yes, my bad